Automattic/jetpack
# jetpack: Remove some tests that were apparently never run

#### Changes proposed in this Pull Request:

These tests were all copied into the repo in #8208 as part of a large
import of code from dops-components. It seems that the testing
infrastructure was never configured to actually run them, and in several
cases necessary supporting files were never copied in.

At this point, 5 years later, it's more likely that someone should
rewrite them from scratch instead of trying to fix these.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Is CI happy?
* You might also check that the various Jetpack test suites still report the same number of tests run as before.
